devtools/gradle/src/main/java/io/quarkus/gradle/QuarkusPlugin.java

```diff
@@ -60,11 +60,13 @@ private void registerTasks(Project project) {
                     quarkusBuild.dependsOn(classesTask);
                 });
 
-        Task buildNative = tasks.create("buildNative", QuarkusNative.class).dependsOn(quarkusBuild);
+        Task buildNative = tasks.create("buildNative", QuarkusNative.class);
```

Comment:
Stupid question but why don't you create the task in the if below?

Reply:
I think one of the Gradle tests would fail. Not 100% certain since I don't have the code in front of me, but I think that was the reason.

Reply:
OK, let's not waste more time on this then :).

Reply:
Yeah the existing Gradle tests are pretty useless I think. Once @gastaldi finishes with the proper integration tests, we should maybe remove them.